lib/widgets/atoms/skill_chip.dart

```diff
@@ -9,7 +9,10 @@ class SkillChip extends StatelessWidget {
   const SkillChip({
     super.key,
     required this.skill,
-    this.icon,
+    this.icon = const Icon(
+      Icons.campaign_outlined,
+      color: Color(0xFF1D192B),
```

Comment:
Avoid hardcoded colors. This will cause issues if, for example, we use dark theme, or just switch to a different color scheme in general.